source/dsl/DangerDSL.ts

```diff
@@ -23,9 +23,8 @@ export interface DangerDSLType {
 export class DangerDSL {
   public readonly github: Readonly<GitHubDSL>
 
-  constructor(pr: GitHubPRDSL, public readonly git: GitDSL) {
-    this.github = {
-      pr
-    }
+  constructor(platformDSL: any, public readonly git: GitDSL) {
+    // As GitLab etc support is added this will need to be changed
+    this.github = platformDSL
   }
```

Comment:
if this will be generic, what about

```counstructor(public readonly platformDSL: Readonly<DSL>, ....) {
}
```

and remove line 24? (basically, rename variable `github` + introduce interface `DSL` for generic instead of any)